docs/ug/cli.md

```diff
@@ -107,7 +107,11 @@ This flag overrides the `Ignore standalone config` field in the CSV config file.
 * Alias: `-l` (lowercase L)
 * Example:`--last-modified-date` or `-l`
 
-The last modified dates will be in the same timezone specified with the `--timezone` flag.
+<box type="info" seamless>
+
+* Cannot be used with ``--shallow-cloning`.
```

Comment:
There are two `` in this line. Needs to be fixed.

Reply:
Fixed